app/Http/Controllers/API/VoucherController.php

```diff
@@ -18,9 +23,10 @@ class VoucherController extends Controller
      *
      * route POST api/vouchers
      *
-     * @param  \Illuminate\Http\Request  $request
-     * @return \Illuminate\Http\Response
+     * @param Request $request
+     * @return ResponseFactory|Application|JsonResponse|\Symfony\Component\HttpFoundation\Response
```

Comment:
IDE verbosity?

Reply:
Or do we really have these return options?

Reply:
Looking upstream, it looks like the response() method is nuts and does actually think it can return those types.

Reply:
Until your IDE got involved, we were happy to umbrella this under `Illuminate\Http\Response` adding here and not everywhere else, might be confusing

Reply:
OK, i'll pull it 😬